src/fed.cpp

```diff
@@ -1782,8 +1781,8 @@ namespace dbm
             return false;
         } else {
             size_t argSize = arg.size();
-            const raw_t* argDBM[argSize];
-            arg.toArray(argDBM);
+            std::vector<const raw_t*> argDBM(argSize);
+            arg.toArray(argDBM.data());
```

Comment:
would it be feasible to rafactor `fed_t::toArray(const raw_t*)` into `std::vector<const raw_t*> fed_t::asVector()` ?